src/lib/Iex/IexMathFpu.h

```diff
@@ -42,7 +42,7 @@ int fpExceptions ();
 //------------------------------------------
 
 extern "C" typedef void (* FpExceptionHandler) (int type, const char explanation[]);
```

Comment:
Does `extern "C"` mean anything for a typedef?

Reply:
I had the same question. It was there, so I just left it, although I don't believe so, but I wasn't sure if it implied "nothrow" when in a C++ context, but is still a C function in a portable fashion?